intelmq/bin/intelmqcli.py

```diff
@@ -424,9 +396,7 @@ def send(self, taxonomy, contact, query, incident_id, requestor=None):
             ziphandle = zipfile.ZipFile(attachment, mode='w',
                                         compression=zipfile.ZIP_DEFLATED)
             data = csvfile.getvalue()
-            if six.PY2:
-                data = unicode(data, 'utf-8')
-            ziphandle.writestr('events.csv', data.encode('utf-8'))
+            ziphandle.writestr('events.csv', data)
```

Comment:
No, according to docs: http://initd.org/psycopg/docs/usage.html#unicode-handling****